### PR TITLE
refactor: move prompt templates to modules which use them

### PR DESCRIPTION
**Pull request by karllu3**

### CODE REVIEW

File: src/dbally/prompts/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from .common_validation_utils import ChatFormat, PromptTemplateError, _check_prompt_variables, _extract_variables
 from .prompt_builder import PromptBuilder
 
-__all__ = ["PromptBuilder"]
+__all__ = ["PromptBuilder", "PromptTemplateError", "_extract_variables", "_check_prompt_variables", "ChatFormat"]
```
Comment (mhordynski) [inline src/dbally/prompts/__init__.py:4]: we shouldn't export private function out of the package, if they're required somewhere else we should change them to public

**Comment by mhordynski** [comment]
I think `refactor:` commit type would be better choice here rather than `fix:`

File: src/dbally/prompts/__init__.py (hunk 1 above)
Comment (karllu3) [reply src/dbally/prompts/__init__.py]: fixed
_extract_variables shall not be exposed
I have changed _check_prompt_variables to check_prompt_variables

**Review by mhordynski — APPROVED** [review]
lgtm 👌

**Pull request merged**